Dockerfile

```diff
@@ -1,28 +1,32 @@
-FROM node:12.16.3-alpine AS builder
+FROM node:16.15.0-alpine as builder
 
-WORKDIR /opt/mojaloop-testing-toolkit
+WORKDIR /opt/app
 
-RUN apk add --no-cache -t build-dependencies git make gcc g++ python libtool autoconf automake \
+RUN apk add --no-cache -t build-dependencies make gcc g++ python3 libtool libressl-dev openssl-dev autoconf automake \
     && cd $(npm root -g)/npm \
     && npm config set unsafe-perm true \
     && npm install -g node-gyp
 
-COPY package.json package-lock.json* /opt/mojaloop-testing-toolkit/
-RUN npm install
-RUN npm install -g bunyan
+COPY package.json package-lock.json* /opt/app/
+RUN npm ci
 
-COPY src /opt/mojaloop-testing-toolkit/src
-COPY spec_files /opt/mojaloop-testing-toolkit/spec_files
-COPY examples /opt/mojaloop-testing-toolkit/examples
-COPY secrets /opt/mojaloop-testing-toolkit/secrets
+COPY src /opt/app/src
+COPY spec_files /opt/app/spec_files
+COPY examples /opt/app/examples
+COPY secrets /opt/app/secrets
```

Comment:
Please remove any secrets from the Docker image itself...it should rather fail if secrets are needed, i.e. someone should be injecting them via a mount

```suggestion
```